core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -165,7 +181,7 @@ class ConsoleConsumerTest extends JUnitSuite {
     propsStream.write("consumer.timeout.ms=1000".getBytes())
```

Comment:
Maybe we want to update this config to use `request.timeout.ms` which is a valid config for the new consumer.